documentation/book/proc-configuring-kafka-mirror-maker-groupid.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-mirror-maker-groupid.adoc
+
+[id='proc-configuring-kafka-mirror-maker-groupid-{context}']
+= Configuring the consumer group identifier
+
+The consumer group identifier, defining the consumer group the consumer for the source Kafka cluster belong to, can be configured using the `groupId` sub-property of `consumer` property in `KafkaMirrorMaker.spec`.
```

Comment:
4 consumers in once sentence!

"The consumer group id can be configured in the `KafkaMirrorMaker.spec.consumer.groupId` property." works just as well imho.